src/AspNet.Security.OAuth.Nextcloud/NextcloudAuthenticationConstants.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AspNet.Security.OAuth.Nextcloud
+{
+    public static class NextcloudAuthenticationConstants
+    {
+        public static class Claims
+        {
+            public const string Groups = "urn:nextcloud:groups";
+            public const string DisplayName = "urn:nextcloud:displayname";
+            public const string Enabled = "urn:nextcloud:enabled";
```

Comment:
Maybe this constant sould be called `IsEnabled` so it's a bit clearer to use from C# code?

Reply:
fix in next commit